search-api/search_api/resources/businesses.py

```diff
@@ -0,0 +1,144 @@
+# Copyright © 2020 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import datetime
+from tempfile import NamedTemporaryFile
+
+from flask import Blueprint, request, jsonify, send_from_directory
+from openpyxl import Workbook
+
+from search_api.auth import jwt, authorized
+from search_api.models import (
+    Corporation,
+    CorpName,
+    Office,
+    _get_corporation_search_results,
+    _merge_corpparty_search_addr_fields,
+    _normalize_addr,
+    _format_office_typ_cd
+)
+
+
+API = Blueprint('BUSINESSES_API', __name__, url_prefix='/api/v1/businesses')
+
+
+@API.route('/search/')
```

Comment:
`/search` doesn't follow the Rest pattern we are following. It needs to be resource based. Just a `GET /businesses `should be enough.